cmd/containerd/command/main_windows.go

```diff
@@ -54,5 +58,36 @@ func handleSignals(ctx context.Context, signals chan os.Signal, serverC chan *se
 			}
 		}
 	}()
+	setupDumpStacks()
 	return done
 }
+
+func setupDumpStacks() {
+	logrus.Debugf("setup dumptacks?")
```

Comment:
s/dumpstacks

Reply:
Lol this should not have been checked in. Reverted and force pushed.